### PR TITLE
update docs for apigateway and other minor changes

### PR DESCRIPTION
**Pull request by csantanapr**
- additions for apigateway for swagger file configuration
- additions to let the user know to use openwhisk npm package
- Other changes for formatting and typos.

### CODE REVIEW

**Comment by csantanapr** [comment]
@mdeuser thanks for the review, please the changes I made base on it 9206dd320415fcaba1d1b3822ac4e1b574de46cc

File: docs/catalog.md [hunk 1]
```diff
@@ -352,7 +352,7 @@ The package includes the following actions.
 
 | Entity | Type | Parameters | Description |
 | --- | --- | --- | --- |
-| `/whisk.system/watson-translator` | package | username, password | Actions for the Watson APIs to translate |
+| `/whisk.system/watson-translator` | package | username, password | Actions for the Watson APIs to translate text and language identification  |
```
Comment (mdeuser) [inline docs/catalog.md]: One last "for the Watson APIs" phrase..

Comment (csantanapr) [inline docs/catalog.md]: good catch, I thought I had them all.

**Comment by csantanapr** [comment]
@mdeuser I think I got them all now, please review this commit  ec6ee1cc6c7099dfecd18f431a0a893b151e3966

**Comment by mdeuser** [comment]
Doc changes LGTM

**Comment by csantanapr** [comment]
@mdeuser I rebase and squash

**Comment by mdeuser** [comment]
@csantanapr - merged

**Pull request merged**